Python/initconfig.c

```diff
@@ -99,9 +103,6 @@ static const char usage_3[] = "\
             when the interpreter displays tracebacks.\n\
          -X frozen_modules=[on|off]: whether or not frozen modules should be used.\n\
             The default is \"on\" (or \"off\" if you are running a local build).\n\
-\n\
---check-hash-based-pycs always|default|never:\n\
-    control how Python invalidates hash-based .pyc files\n\
 ";
 static const char usage_4[] = "\
```

Comment:
I don’t know why the help text is `split into parts < 512 bytes` (comment on line 31); usage_xoptions (was usage_3) doesn’t seem to be, and it has worked fine.